crates/cargo-test-support/src/registry.rs

```diff
@@ -451,39 +451,44 @@ impl RegistryBuilder {
 ///
 /// # Example
 /// ```
-/// // Publish package "a" depending on "b".
-/// Package::new("a", "1.0.0")
-///     .dep("b", "1.0.0")
-///     .file("src/lib.rs", r#"
-///         extern crate b;
-///         pub fn f() -> i32 { b::f() * 2 }
-///     "#)
-///     .publish();
+/// use cargo_test_support::registry::Package;
+/// use cargo_test_support::project;
 ///
-/// // Publish package "b".
-/// Package::new("b", "1.0.0")
-///     .file("src/lib.rs", r#"
-///         pub fn f() -> i32 { 12 }
-///     "#)
-///     .publish();
+/// fn crate_package_test() {
```

Comment:
Instead of adding a function, I think either adding a `no_run` attribute to the code block, or importing and annotating with `cargo_test` macro is a bit better.